### PR TITLE
tmpl tests fixes

### PR DESCRIPTION
**Pull request by JakubMatejka**
viz. https://keboola.slack.com/archives/C02JAQXLY92/p1666009677145889

### CODE REVIEW

File: internal/pkg/template/template.go [hunk 1]
```diff
@@ -37,6 +37,7 @@ const (
 	ReadmeFile           = "README.md"
 	SrcDirectory         = "src"
 	TestsDirectory       = "tests"
+	InstanceId           = "instance-id"
```
Comment (michaljurecko) [inline internal/pkg/template/template.go:40]: Mohol by si to spravi prosim nejake jasnejsie `InstanceIdForTest`? , alebo pridat komentar.

**Review by michaljurecko — APPROVED** [review]
super, iba jedna drobnost

File: internal/pkg/template/template.go (hunk 1 above)
Comment (michaljurecko) [reply internal/pkg/template/template.go:40]: Este mi napadlo, ze by si to mohol pouzit v e2e testoch ako monika, v `name` aby sme si boli isty, ze to funguje (`InstanceIdShort`).

Comment (JakubMatejka) [reply internal/pkg/template/template.go:40]: jo, pravda 👍

Comment (JakubMatejka) [reply internal/pkg/template/template.go]: https://github.com/keboola/keboola-as-code/pull/894/commits/64e1e8e28d8f4394f1b88ba612faef41fbcf1f33

**Pull request merged**